Smithsonian/ForestGEO
# PageMenu: Rename SelectedMenu to PageMenu, folderize, make responsive

- [x] Rename to PageMenu, as it's a bit more descriptive
- [x] Move into folder
- [x] Add a PageMenu.stories.tsx
- [x] Use Button, for tab navigation and on hover mouse
- [x] Use the path location so if route is selected by URL it selects the right one
- [x] Make responsive, so menu is shown on mobile, and buttons are shown on desktop. See [mui responsive app bar docs](https://mui.com/material-ui/react-app-bar/#app-bar-with-responsive-menu).

### How to use/test?

- See PageMenu in the storybook
```bash
cd FrontEnd
npm run storybook
```

- With a very narrow browser window (or using mobile dev tools) the menu should be active.
- In desktop there should be three links in the menu bar.
- In the static web app link, go to /report in the URL bar of the browser and the selected menu should be "Report".
